doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -132,14 +132,13 @@ Cabinet is the rack holder for the server and network devices. The data will be
 
 <a name="5.3.4"></a>
 ### 5.3.4 Server
-Server device data defines the key information about how to access the server remotely, i.e. remote ip, user, credentials.
-There are many ways to validate the hardware by remote interface verifcation, for example IPMI,redfish etc.
-Considering the amount of server, user can validate the server interface in distributed way or centralized way.
-Distributed validation is to temporally install OS with inband ip configured, which can connect itself remote ip, and invoked the remote(BMC,IPMI) ip and validate the inteface from OS.
-Centralized validation is to invoke and validate the interface from remote access interface.
-This is totally decided by operator theirselves.
-For storage type server, centralized way proposed as the amount is less than computational type server.
-For computational type server it's better to choose distributed validation way.
+The server device data defines key information about how to access the server remotely, for example remote IP address, user, credentials and so on.
+There are many ways to validate the hardware by remote interface verification, for example IPMI, redfish and so on.
+Considering the number of servers, the user can choose to validate the server interface in a distributed way or in a centralized way.
```

Comment:
do you want to state that distributed is network connected (e.g., LAN) and centralised uses a direct connection

Reply:
Both of them are network connected. The difference is that in case of a distributed validation, the validation is initiated by the servers. In case of centralized validation, it is initiated centrally.

Reply:
@EditKoselak Then the sentences should be worded to state that. As currently written it appears that for distributed validation, an IP address is assigned, etc. but not so for for centralised validation.

When you say, "it is initiated centrally" what does that mean?

Reply:
@pgoyal01 Thanks for the comment! We have removed the part about the operating system and the IP address. Does this clarify the description?